### PR TITLE
[ws-scaler] Be less loud when ws-manager is unavailable

### PR DESCRIPTION
**Pull request by csweichel**
This PR tones down ws-scheduler when ws-manager is not available and ws-scaler cannot work.
It also makes the scaler more stable by avoiding a race between ghost renewal and regular scheduling.

### How to test
1. `kubectl scale deployment --replicas=0 ws-manager` ... you should see `ws-manager is unavailable` in the ws-scheduler logs
2. `kubectl scale deployment --replicas=1 ws-manager` ... you should see `status update` in the ws-scheduler logs

### CODE REVIEW

**Review by akosyakov — APPROVED** [review]
I checked that it works as advertized. But it would be good if someone looks over changed code cc @geropl

File: components/ee/ws-scheduler/pkg/scaler/driver.go [hunk 1]
```diff
@@ -162,38 +167,33 @@ func (wspd *WorkspaceManagerPrescaleDriver) Run() {
...
+				// candidate to be selected. If we left it at that, we might never renew anything, hence forcing
+				// the renewal of at least one candidate.
+				d = 1
 			}
 			if d > len(status.DeletionCandidates) {
 				d = len(status.DeletionCandidates)
 			}
-			log.WithField("count", d).Info("attempting to renew ghost workspaces")
+			// we're only deleting ghosts here, not starting any. In the next scheduling interval, we'll restart
+			// the neccesary ghosts.
```
Comment (geropl) [inline components/ee/ws-scheduler/pkg/scaler/driver.go:185]: :100: :+1:

File: components/ee/ws-scheduler/pkg/scaler/driver.go [hunk 2]
```diff
@@ -205,9 +205,10 @@ func (wspd *WorkspaceManagerPrescaleDriver) Run() {
 					d = len(status.DeletionCandidates)
 				}
 				err = wspd.stopGhostWorkspaces(ctx, status.DeletionCandidates[:d])
-			}
-			if d > 0 {
+			} else if d > 0 {
```
Comment (geropl) [inline components/ee/ws-scheduler/pkg/scaler/driver.go:208]: :heart:

**Review by geropl — APPROVED** [review]
:heavy_check_mark:

**Pull request merged**